splash/qtrender_lua.py

```diff
@@ -328,7 +329,7 @@ def __call__(self, *args):
             args=escape_js(*args)
         )
         wrapper_script = get_process_errors_js(expr)
-        res = self.tab.evaljs(wrapper_script)
+        res = self.tab.evaljs(wrapper_script, result_protection=self.js_result_protection)
```

Comment:
If you turn on `result_protection` some objects cannot be retreived, for example `getBoundaryRect()` returns an instance of [DOMRect](https://developer.mozilla.org/en-US/docs/Web/API/DOMRect) which is ignored in sanitize function.